.github/workflows/ci.yaml

```diff
@@ -4,11 +4,11 @@ on:
     types: [opened, synchronize, reopened]
   push:
     branches:
-    - master
+    - '*'
+    - '!dependabot/**'
```

Comment:
Might as well run these for all branches, excluding dependabot because dependabot created branches do not have access to repo secrets.